README.md

```diff
@@ -12,16 +12,17 @@
-| [mux-video](packages/mux-video)                   | A Mux-flavored HTML5 video element            | Web Component | Public Beta             |
-| [mux-video-react](packages/mux-video-react)       | A Mux-flavored React video component          | React         | Public Beta             |
-| [mux-audio](packages/mux-audio)                   | A Mux-flavored HTML5 audio element            | Web Component | Public Beta             |
-| [mux-audio-react](packages/mux-audio-react)       | A Mux-flavored HTML5 audio component          | React         | Public Beta             |
-| [mux-player](packages/mux-player)                 | A Mux-flavored HTML5 media player component   | Web Component | 1.x.x 🎉                |
-| [mux-player-react](packages/mux-player-react)     | A Mux-flavored React media player component   | React         | 1.x.x 🎉                |
-| [mux-uploader](packages/mux-uploader)             | A Mux-flavored HTML5 media uploader component | Web Component | Public Beta             |
-| [mux-uploader-react](packages/mux-uploader-react) | A Mux-flavored React media uploader component | React         | Public Beta             |
+| Name                                                        | Description                                   | Framework     | Official Release Status |
+| ----------------------------------------------------------- | --------------------------------------------- | ------------- | ----------------------- |
+| [mux-video](packages/mux-video)                             | A Mux-flavored HTML5 video element            | Web Component | Public Beta             |
+| [mux-video-react](packages/mux-video-react)                 | A Mux-flavored React video component          | React         | Public Beta             |
+| [mux-audio](packages/mux-audio)                             | A Mux-flavored HTML5 audio element            | Web Component | Public Beta             |
+| [mux-audio-react](packages/mux-audio-react)                 | A Mux-flavored HTML5 audio component          | React         | Public Beta             |
+| [mux-player](packages/mux-player)                           | A Mux-flavored HTML5 media player component   | Web Component | 1.x.x 🎉                |
+| [mux-player-react](packages/mux-player-react)               | A Mux-flavored React media player component   | React         | 1.x.x 🎉                |
+| [mux-uploader](packages/mux-uploader)                       | A Mux-flavored HTML5 media uploader component | Web Component | Public Beta             |
+| [mux-uploader-react](packages/mux-uploader-react)           | A Mux-flavored React media uploader component | React         | Public Beta             |
+| [mux-active-viewer-count](packages/mux-active-viewer-count) | A Mux-flavored HTML5 viewer count component   | Web Component | Public Alpha            |
```

Comment:
I wonder if `HTML5` is needed here. Above, I believe it's there modifying the `video` and `audio` elements.

Reply:
I get what you're saying, but it's at least currently also applied to `mux-uploader`:

> A Mux-flavored HTML5 media uploader component

I'd propose we keep it for now for consistency.

Reply:
Leaving for now, for the sake of consistency